core/plugins/org.polarsys.capella.core.data.migration/src/org/polarsys/capella/core/data/migration/contribution/ImagePathInRichTextAttributeContribution.java

```diff
@@ -300,7 +336,7 @@ private Optional<File> getFileFromString(String fileString) {
    * Get the image file to create in the images folder of the project.
    */
   public IFile getFileToCreate(IFolder imageFolder, File fileToCopy, EObject contextObject) {
-    String wsImageName = ImageManager.IMAGE_FOLDER_NAME + File.separator + fileToCopy.getName();
+    String wsImageName = this.imagesFolderName + File.separator + fileToCopy.getName();
     IFile targetImageFile = imageFolder.getProject().getFile(wsImageName);
 
     return targetImageFile;
```

Comment:
replace by 
imageFolder.getFile(fileToCopy.getName())

so you can delete this method i think

Reply:
done